wordle-web/src/views/game.vue

```diff
@@ -0,0 +1,56 @@
+
+
+<template>
+  <div class="game">
+    <VCard variant="tonal">
+      <VCardTitle>
+        WordleCopy
+      </VCardTitle>
+      <VCardSubtitle>
+        Sample Text. Lorem ipsum dolor sit amet consectetur adipisicing elit. Numquam, fuga consectetur.
+      </VCardSubtitle>
+      <v-divider :thickness="5"></v-divider>
+      <v-container>
```

Comment:
Typically, containers are only used at the page level to control page whitespace on different sized screens.